### PR TITLE
update the guide for modifying external user permission on aws console

### PR DESCRIPTION
**Pull request by timckt**

### CODE REVIEW

File: runbooks/source/aws-access-control.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+9) Verify the user is created. (You can use AWS Console for this.)
+
+10) Tell them they can login here: https://aws-login.cloud-platform.service.justice.gov.uk
+
+## Activating MFA for new users
+
+Unfortunataly terraform can't activate MFA for users, this process must be done done manually either [through AWS Console (UI)](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa_enable_virtual.html) or [through the AWS CLI](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_mfa_enable_cliapi.html).
+
+## Modifying external user permissions
```
Comment (poornima-krishnasamy) [inline runbooks/source/aws-access-control.html.md.erb:46]: can we rename that as service teams or cloud-platform users. External reads a bit outside of moj

File: runbooks/source/aws-access-control.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+This part is the guideline for hanlding requests arise to modify AWS resource permissions on AWS console for users outside the Cloud Platform Team, such as granting read access to ElastiCache for external user.
```
Comment (poornima-krishnasamy) [inline runbooks/source/aws-access-control.html.md.erb:48]:
```suggestion
This part is the guideline for handling requests arise to add or modify read only access to any aws resources that are created for Cloud Platform users.
```

File: runbooks/source/aws-access-control.html.md.erb (hunk 1 above)
Comment (timckt) [reply runbooks/source/aws-access-control.html.md.erb]: updated with more appropriate wording

File: runbooks/source/aws-access-control.html.md.erb (hunk 2 above)
Comment (timckt) [reply runbooks/source/aws-access-control.html.md.erb]: updated with more appropriate wording

**Review by poornima-krishnasamy — APPROVED** [review]

**Pull request merged**